EXILED/Exiled.API/Features/Roles/Scp3114Role.cs

```diff
@@ -160,7 +160,7 @@ public RoleTypeId StolenRole
         /// <summary>
         /// Gets or sets the SCP-3114's Ragdoll used for it's FakeIdentity.
         /// </summary>
-        public Ragdoll Ragdoll
+        public new Ragdoll Ragdoll
```

Comment:
// TODO: Change Name FakeRagdoll or something else than Ragdoll (because they don't have the same purpose but doing it would make breaking change